tools/packaging/kernel/configs/fragments/x86_64/sev/sev.conf

```diff
@@ -4,8 +4,7 @@ CONFIG_AMD_MEM_ENCRYPT=y
 CONFIG_EFI=y
 CONFIG_EFI_SECRET=m
 CONFIG_EFI_STUB=y
-CONFIG_MODULE_SIG=y
```

Comment:
I dont know if this config is needed, if so, it will need the same dockerfile addition as from 
[unmesh's sev changes](https://github.com/kata-containers/kata-containers/pull/6591/files#diff-289f4440b4dcbe7b0cc0c54b3a4ce64607dd0c8ebd8d3a4b1014a29f72487a18)